src/api-service/__app__/onefuzzlib/azure/nsg.py

```diff
@@ -39,7 +39,7 @@ def get_nsg(name: str) -> Optional[NetworkSecurityGroup]:
         nsg = network_client.network_security_groups.get(resource_group, name)
         return cast(NetworkSecurityGroup, nsg)
     except (ResourceNotFoundError, CloudError) as err:
-        logging.error("nsg %s does not exist: %s", name, err)
+        logging.debug("nsg %s does not exist: %s", name, err)
```

Comment:
do we even need to log this ? it seems like the return type already indicates that we expect that it might not exist

Reply:
just in case. we can remove it later